### DISCUSSION TITLE
Local host login password coming up as incorrect

### DISCUSSION THREAD

**Discussion by AlphaDeltaGamma** [Security-Onion-Solutions/securityonion#11668] [General]
### Version
  2.4.20
### Installation Method
Security Onion ISO image
### Location
on-prem with Internet access
### Ram 
16
### Storage for
250 gb

Hi everyone
i seem to have a problem while installing sec onion
The password is coming up as incorrect even though I'm typing the correct password.

![Screenshot (74)](https://github.com/Security-Onion-Solutions/securityonion/assets/92504746/40bc7bd6-04c8-411e-8514-ec82717d3561)

**Comment by dougburks** [comment]
Above you say that the version is 2.4.20 but looking at the screenshot, it appears you are trying to login to an old 2.3 installation. Make sure that you are performing a new installation of 2.4.20 using our Security Onion ISO image. If this VM had a previous 2.3 installation you might want to create a new VM altogether.
